ambry-api/src/main/java/com/github/ambry/router/GetBlobResult.java

```diff
@@ -23,15 +25,18 @@
 public class GetBlobResult {
   private final BlobInfo blobInfo;
   private final ReadableStreamChannel blobDataChannel;
+  private final List<StoreKey> blobChunkIds;
```

Comment:
we already have a class called GetBlobResultInternal, which includes List of store keys. Though that serves as an internal structure, we should probably remove that class all together now that we have this one.

Reply:
You are right, I removed the GetBlobResultInternal.